zerver/openapi/zulip.yaml

```diff
@@ -19456,6 +19456,10 @@ paths:
                             description: |
                               Whether the user can authenticate using their Microsoft Entra ID account.
                             type: boolean
+                          azureadb2c:
+                            description: |
+                              Whether the user can authenticate using their Microsoft Entra B2C ID account.
+                            type: boolean
```

Comment:
This needs to be documented as new with an API feature level bump. `tools/create-api-changelog` may help.